src/material/autocomplete/autocomplete-trigger.ts

```diff
@@ -879,13 +882,35 @@ export class MatAutocompleteTrigger
     });
   }
 
+  /** Implements BreakpointObserver to be used to detect handset landscape */
+  private _breakpointObserver = inject(BreakpointObserver);
+
   private _getOverlayPosition(): PositionStrategy {
+    // Set default Overlay Position
     const strategy = this._overlay
       .position()
       .flexibleConnectedTo(this._getConnectedElement())
       .withFlexibleDimensions(false)
       .withPush(false);
 
+    if (!this._handsetLandscapeBreakpointSubscription) {
```

Comment:
This is kind of a weird place to subscribe (as a side-effect in a getter function) can you move it to `_attachOverlay` right after we create `_viewportSubscription`